src/oc/hoc.cpp

```diff
@@ -1645,7 +1643,6 @@ int hoc_yyparse(void) {
 		switch (i) {
         case 'e':
             i = '\n';
-            hoc_edit();
             break;
```

Comment:
I think this switch is useless now

Reply:
Yes, we no longer have an inbuilt editing option.